src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+      "        computation. See http://www.hsl.rl.ac.uk.\n",
+      "******************************************************************************\n",
+      "\n",
+      "This is Ipopt version 3.13.2, running with linear solver ma27.\n",
+      "\n",
+      "Number of nonzeros in equality constraint Jacobian...:        5\n",
+      "Number of nonzeros in inequality constraint Jacobian.:        0\n",
+      "Number of nonzeros in Lagrangian Hessian.............:        2\n",
+      "\n",
+      "Total number of variables............................:        3\n",
+      "                     variables with only lower bounds:        0\n",
+      "                variables with lower and upper bounds:        2\n",
+      "                     variables with only upper bounds:        0\n",
+      "Total number of equality constraints.................:        3\n",
+      "Total number of inequality constraints...............:        0\n",
+      "        inequality constraints with only lower bounds:        0\n",
+      "   inequality constraints with lower and upper bounds:        0\n",
+      "        inequality constraints with only upper bounds:        0\n",
+      "\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "   0  0.0000000e+00 1.42e-14 0.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "\n",
+      "Number of Iterations....: 0\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Constraint violation....:   1.4210854715202004e-14    1.4210854715202004e-14\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   1.4210854715202004e-14    1.4210854715202004e-14\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 1\n",
+      "Number of objective gradient evaluations             = 1\n",
+      "Number of equality constraint evaluations            = 1\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 1\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 0\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.001\n",
+      "Total CPU secs in NLP function evaluations           =      0.000\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n",
+      "\b\b\b\b\b\b\b\b\b\b\b\b\b\b"
+     ]
+    }
+   ],
+   "source": [
+    "# Solve the simulation using ipopt\n",
+    "# Note: If the degrees of freedom = 0, we have a square problem\n",
+    "opt = SolverFactory('ipopt')\n",
```

Comment:
We should use the IDAES `get_solver` method instead of the Pyomo solver factory.